WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController+Extensions.swift

```diff
@@ -246,7 +334,19 @@ private extension NotificationsViewController
         return WordPressAppDelegate.sharedInstance().simperium
     }
 
-    struct RatingSettings {
+    enum Properties {
+        static let sortKey          = "timestamp"
+    }
+
+    enum Filter: Int {
+        case None                   = 0
+        case Unread                 = 1
+        case Comment                = 2
+        case Follow                 = 3
+        case Like                   = 4
+    }
+
+    enum RatingSettings {
```

Comment:
Curious about the switch to `enum` from `struct`. A `struct` seems better since there are no `case`s.